mix/qml/html/codeeditor.js

```diff
@@ -199,3 +189,5 @@ compilationComplete = function()
 	}
 	compilationCompleteBool = true;
 }
+
+editor.setOption("extraKeys", extraKeys);
```

Comment:
This works because CM keeps reference to extraKeys object, so it can be modified at any time